.gitignore

```diff
@@ -16,3 +16,4 @@ bin/sabre-cs-fixer*
 
 # development stuff
 .php_cs.cache
+.idea
```

Comment:
This folder should be ignored via your global git config

Reply:
I've fixed